src/_scss/pages/search/searchSidebar.scss

```diff
@@ -4,6 +4,17 @@
     @import "./_filters";
     @import '../../elements/_inputs';
     @import '../../elements/_labels';
+
+    .sidebar-bottom__keyword-link-container{
+        @media(min-width:$medium-screen) {
+            display: none;
```

Comment:
It's better to detect the screen size in the jsx file and show or not show the component there, than to use display: none, because display: none leaves the element in the dom, and hides it. But it doesn't need to be in the dom at all if it isn't on the page in most cases.